src/Microsoft.AspNetCore.BuildTools.ApiCheck/Microsoft.AspNetCore.BuildTools.ApiCheck.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+  <Import Project="..\..\build\common.props" />
+  <PropertyGroup>
+    <VersionPrefix>1.0.1</VersionPrefix>
```

Comment:
Will do this in a follow PR along with stopping the cross-compilation. Primary goal of this PR was to preserve existing build input/output but w/ csproj.